language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -82,6 +83,12 @@ tReset withSandbox = testCase "reset" $ run withSandbox $ \_ -> do
     lid2 <- getLedgerIdentity
     liftIO $ assertBool "lid1 /= lid2" (lid1 /= lid2)
 
+tMultipleResets :: SandboxTest
+tMultipleResets withSandbox = testCase "multipleResets" $ run withSandbox $ \_pid -> do
+    forM_ [1 .. 20 :: Integer] $ \_ -> do
+        lid <- getLedgerIdentity
+        Ledger.reset lid
```

Comment:
Is there any expected result here we could test for? Or are we just happy if all of this runs through without any error?

Reply:
I like tests too. And I am happy for this test to be added.

But I just wanted to make it clear, and hope we all agree :

- This new test is not really a test of the haskell ledger bindings.
- It's not really even an integration test of the sandbox ledger and the haskell bindings.
- What it is, is a stress test of the sandbox ledger, making use of the haskell ledger bindings.

Reply:
@hurryabit We're happy if it runs without an error, but I extended it to verify uniqueness of received ledger IDs as we have these at hand anyway.

@nickchapman-da I'd say it actually is an integration test - we use gRPC for integration and this test starts failing after gRPC upgrade on Windows, while it works ok on current master. It might be caused by sth. specific to sandbox part, but not necessarily.